frontend/src/components/DocCardBadges.tsx

```diff
@@ -32,35 +32,38 @@ const DocCardBadges: React.FC<DocCardBadgesProps> = ({ odhDoc }) => {
     }
   }, [qsContext, docType, docName]);
 
-  const label = DOC_TYPE_LABEL[docType] || 'Documentation';
+  if (docType !== null) {
+    const label = DOC_TYPE_LABEL[docType] || 'Documentation';
 
-  return (
-    <LabelGroup defaultIsOpen numLabels={3}>
-      <Tooltip content={DOC_TYPE_TOOLTIPS[docType]}>
-        <Label color={getLabelColorForDocType(docType)}>{label}</Label>
-      </Tooltip>
-      {duration ? (
-        <Label variant="outline" color="grey">
-          {getDuration(duration)}
-        </Label>
-      ) : null}
-      {completionStatus === CompletionStatusEnum.InProgress ? (
-        <Label variant="outline" color="purple" icon={<SyncAltIcon />}>
-          In Progress
-        </Label>
-      ) : null}
-      {completionStatus === CompletionStatusEnum.Success ? (
-        <Label variant="outline" color="green" icon={<CheckCircleIcon />}>
-          Complete
-        </Label>
-      ) : null}
-      {completionStatus === CompletionStatusEnum.Failed ? (
-        <Label variant="outline" color="red" icon={<ExclamationCircleIcon />}>
-          Failed
-        </Label>
-      ) : null}
-    </LabelGroup>
-  );
+    return (
+      <LabelGroup defaultIsOpen numLabels={3}>
+        <Tooltip content={DOC_TYPE_TOOLTIPS[docType]}>
+          <Label color={getLabelColorForDocType(docType)}>{label}</Label>
+        </Tooltip>
```

Comment:
This is the only node that should be conditionally rendered against the `docType` and not the entire block.